SITE_CONTRIBUTION.md

```diff
@@ -0,0 +1,165 @@
+Spaces within a heading are replaced with a hyphen. `## My heading` has a link `#My-heading`.
+
+**Linking to a section within another page:**
+
+Unfortunately, the `{% link %}` templating method does not allow linking to a section within a page. Use this format when linking to a section within another page.
+`{% <Path to page post generation/<section> %}` or `/docs/v1.0.0/deploy-options/#Host-Networking`
+
+A key difference between using `{% link %}` and the above method is that `{% link %}` will resolve to pages final location. To use the above method, you need to point the link to the location where the document will end up. For example, files in the `_guides` directory will be located in `/guides` on the live site. Therefore, the link for `_guides/cert-manager` is `/guides/cert-manager`.
+
+Additionally, the above method will not follow any permalink changes or page-specific permalink.
+
+**Linking to the project contour GitHub repo:**
+
+The GitHub metadata plugin populates Jekyll variables with data from GitHub during build. When inserting a link to the Project Contour repository, use the prefix `{{site.github.repository_url}}`.
+
+A full list of variables can be found in the [jekyll-github-metadata documentation][4].
+
+**Linking to an external page:**
+
+Links to external pages are done using the full URL, `[1]: https://projectcontour.io` for example.
+
+## Proofread and spell check
+
+If you're making any content submissions, please proofread and spell check your work.
+
+Spell check can be run using `make check`.
+
+## Local Site Testing
+
+Perform local tests before creating a PR to detect issues such as broken links.
+
+**Using docker:**
+
+The easiest way to run the site locally is by using the top-level [Makefile](../Makefile). This has a `site-devel` target and runs a [Jekyll](https://jekyllrb.com) development server in Docker, which is useful for checking local documentation changes. 
+By default, the site will be available on [port 4000](http://127.0.0.1:4000/).
+
+Note that the version of [Jekyll](https://jekyllrb.com) used by the site is pinned in the [Gemfile](./Gemfile.lock). The `site-devel` target uses the matching versioned Docker image to ensure compatibility.
+
+Once you are happy with your changes, commit them and push everything
+to your fork.  When you submit a PR, Netlify will automatically
+generate a preview of your changes.
+
+**Manual Jekyll Instance:**
+
+If you need to manually install and run [Jekyll](https://jekyllrb.com)
+without using Docker, you can install the [Jekyll](https://jekyllrb.com)
+packages on your development system.
+
+On macOS, install the following:
+
+- `brew install rbenv`
+- `rbenv install 2.6.3`
+- `gem install bundler`
+
+On Ubuntu you will need these packages:
+
+- ruby
+- ruby-dev
+- ruby-bundler
+- build-essential
+- zlib1g-dev
+- nginx (or apache2)
+
+Install [Jekyll](https://jekyllrb.com) and plug-ins in one fell
```

Comment:
nit: it seems inconsistent that this doesn't use footnote style links

Reply:
Good pickup